src/encoded/tests/test_search.py

```diff
@@ -596,7 +671,49 @@ def test_index_data_workbook(app, workbook, testapp, indexer_testapp, htmltestap
                 html_res = htmltestapp.get(item_res['@id'])
                 assert html_res.body.startswith(b'<!DOCTYPE html>')
             except Exception as e:
+                ignored(e)
                 pass
+    if ItemTypeChecker.CONSIDER_DELETED:
+        print(f"(CONSIDER_DELETED) Items deleted = {ItemTypeChecker.DELETED_SEEN}")
+        print(f"db_total={db_total} es_total={es_total} deleted_seen={ItemTypeChecker.deleted_seen()}")
+        assert(db_total == es_total + ItemTypeChecker.deleted_seen())  # 2nd is db, 4th is es
```

Comment:
Here we accommodate deleted items. They will be present in situations where `ItemTypeChecker.DELETED_ITEMS` is `True`, so we use that as a conditional.  (Some tests bind this class variable using `with local_attrs(): ...`)